pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1527,6 +1527,43 @@ public void resetCursor(
         }
     }
 
+    @POST
+    @Path("/{tenant}/{namespace}/{topic}/subscription/{subName}/updateProperties")
```

Comment:
```suggestion
    @PUT
    @Path("/{tenant}/{namespace}/{topic}/subscription/{subName}/properties")
```

It would be more RESTful to model the properties as a resource that can be updated with `PUT`. 
It would also be RESTful to make a GET return the existing properties.

Reply:
I have switched to PUT, thanks
for GET I would prefer to do a follow up PR, as the information is already available in stats and I would prefer to keep the patch as small as possible